### ISSUE TITLE
Set.Range( "B2:E9", seq(1,10))

### ISSUE THREAD

**Issue by alexlangnau** [sdllc/Basic-Excel-R-Toolkit#17]
when I try: Set.Range( "B2:E9", seq(1,10))

I get a 8X4 Matrix as output containing only 1's; 

it looks like:

![capture](https://cloud.githubusercontent.com/assets/20053612/16211326/207b4c4e-3741-11e6-9599-356434c16dce.PNG)

what am I doing wrong?

thanks

**Comment by duncanwerner** [comment]
We introduced a bug in the last release that's causing this.  Three things you can do:

(1) there's an older function which works, called "Set.Range.1", so 

```
Set.Range.1( "b2:e9", seq( 1, 10 ))
```

(2) use the Excel scripting API:

```
rng <- EXCEL$Application$get_Range( "b2:e9" )
rng$put_Value( matrix( seq( 1, 10 ), nrow=8 ))
```
(3) use the new release, which I am testing now and which should be ready soon.

**Comment by duncanwerner** [comment]
Fixed in v0.73
